### ISSUE TITLE
When will be release of vestacp 0.9.8-18? Where is roadmap? How many work left?

### ISSUE THREAD

**Issue by anacondaqq** [serghey-rodin/vesta#1215]
### Problem:
[0.9.8-17 is Released on Friday Nov 25, 2016](https://vestacp.com/roadmap/). It's more then 8 months already. Eight months without any updates it's very big period of time. I can see some activity at branch, but i don't see the sense of these changes, because i don't see any defined goals.

### List of questions to be answered
1. When will be released new version of vestacp? 
2. If you have plan for vestacp 0.9.8-18 version, can you share it? I mean what things must be fixed, and what issues must be closed, cleaned, solved, etc. If you don't have it for opensource project - it's a big fault. 
3. How can community (for example me) help you to finish new version of vestaCP?

### From common phrases to concrete actions
1. Define things what must be fixed, things what must be updated to actual versions of software stack.
2. Print issues here for example to issue tracker, and create boards
3. Create board called "0.9.8-18 version', and place all tasks what must be finished until you will be ready for releases
4. Ask your community to help you to finish all of  these tasks.
5. People will help you in very short time period to finish all of the issues
6. In result we will receive up to date and actual VestaCP with latest software which includes latest updates, with fixed reported bugs.

### What we have now
9 months old CP, with very old software stack with different security issues inside. These issues can be fixed  by user, but for what then need control panel? (rhetorical question)

### All what i wish by opening the issue:
1. Organizationality, and concentration.
2. Understanding how much work is done, and how much is left.
3. Understanding how I can help you, and also how the community will be able to help you, in order to release a finished product for the masses, and update the already outdated current version of VestaCP

Thank you for your attention.

**Comment by peterbrinck** [comment]
They have a roadmap, but it seems to be down (again......)
When it's online, it's available here: https://bugs.vestacp.com/

**Comment by madeITBelgium** [comment]
Hi @anacondaqq 

We are working on the release. I cannot give you a date.
At this moment the devs are working on the Softaculous integration. When that is done we need to test the control panel. The problem is that we do not have tests for the most parts of the control panel. So everything is tested manually and that is not easy. The testing is taking a lot of time and energy. So If you wanna help? Test the software.

We need to change our testing before we can make more release, we can not make a release without testing. 

You can check the roadmap here: https://vestacp.com/roadmap/

**Comment by anacondaqq** [comment]
@madeITBelgium Do you have any private contacts? I can help with testing.

**Comment by blockademarc** [comment]
Hi @madeITBelgium ,
I'm also interested to help testing - Please write me a short message how to reach you.
Thanks.

**Comment by n1trux** [comment]
I guess he means unit tests, not actual human tests?

**Comment by kevit-development** [comment]
@madeITBelgium I think the dev's should work on bug fixes (closing issues on GitHub issue tracker), fixing security issues and updating older packages.

Who even cares about Softaculous intergration?

**Comment by peterbrinck** [comment]
I actually agrees with you, @kevit-development 

Softaculous more often than not installs outdated and/or even broken installations and apps.
At least I hope that it's going to be an optional thing to include in the Vesta installation

**Comment by kevit-development** [comment]
Additional comment to https://github.com/serghey-rodin/vesta/issues/1215#issuecomment-311646589

> @madeITBelgium https://github.com/serghey-rodin/vesta/issues/1215#issuecomment-309283431: We are working on the release. I cannot give you a date.
  
As @anacondaqq https://github.com/serghey-rodin/vesta/issues/1215#issue-236696614 said, the latest version 0.9.8-17 released on 25th of November 2016. Thats a long time ago. Roadmap said the 0.9.8-18 release was "scheduled" on (end) March 2017. It's June/July now...

> @madeITBelgium https://github.com/serghey-rodin/vesta/issues/1215#issuecomment-309283431: At this moment the devs are working on the Softaculous integration.

As I mentioned above, why is all the development time being invested in some 3rd party intergration, I don't think more than 10% of the VestaCP users / systemadmins will ever use it. Focus on the important stuff, that I am sure 100% of all the VestaCP users / systemadmins want: 1) **Security**, and 2) **Stability**.

TLDR; Development focus should be put on security and stability.

**Comment by yokowasis** [comment]
I totally agree the dev should focus and security and stability. But on the other hand, the end user (shared hosting user), need an easy way to install script on their website. I am not really a fan of softaculous. I prefer the devs should create its own App Installer, it should be easy. much more easier than integrating softaculous into vestacp.

**Comment by dpeca** [comment]
in 99% cases users needs WordPress, installer for WP can be writen in 10 bash lines... few javascript lines, minimum admin panel HTML template changes...
code can be written in one hour...

**Comment by yokowasis** [comment]
@dpeca that's what I think. Instead go full blown softaculous, just create a few essential app script. Other user on github can improve / add another not so common app installer script. Softaculous is so overrated. It's not like we install wordpress everyday.

**Comment by anton-reutov** [comment]
Now we are testing Softaculous integration with Softaculous team. There are many aspects which need to be solved

**Comment by n1trux** [comment]
I think softaculous is a great way of doing things because it's kind of a [standard](https://xkcd.com/927/).

If you think vestaCP is developing too slowly or softaculous is not the way to go, feel free to fork this and do it better. But don't tell the devs what to do, this is a Free and Open Source Software product, not something you buy.

You can actually hire a developer for real money to implement new features in vestaCP and you might even get the PR upstream. But stop trying to be project managers for people you don't pay.

**Comment by peterbrinck** [comment]
I agree with you @n1trux for the most part.
But WHEN it's an open-source project, people may not contribute with code but in other ways, like this.
By expressing their concerns, both good and bad.
That's part of the open-source as well, and I think this is often forgotten.

But in anyways, I think that a simple installer for eg. WordPress would be much nicer than Softacolous, since it has taken the developers so long to finish it and develop it, and I think that the focus should be on more need-to-have things rather than nice-to-have, which Softacolous is.

But I'm glad to hear that they are testing it with the Softacolous team!
And I still think the developers are doing a great job!

**Comment by yokowasis** [comment]
it's called feedback / suggestion. If we don't tell the devs what to do, the devs won't have any idea what should be done, and what shouldn't be done. The Bug tracker, and the Issues is made specifically for this.

"Hey dev, I found a bug / issue, you should fix it"

Softaculous is nice to have. But not that important. Most People will just use it to install wordpress. Not to mention we have to pay the license.

**Comment by huloza** [comment]
Yes guys, please focus on security and stability, softaculous is not that important.

We are here to help and we want to keep this alive.

regards!

**Comment by xjlin0** [comment]
I do care about Softaculous integration.   Clients  and non-tech users do need them.  Without clients we don't even need server control panel such as CPanel or VestCP.

**Comment by huloza** [comment]
guys what do you thing if we fork this, apply some of the pull request(specially the urgents), fix the real issues, and do the testing, so when the developers comeback(if they do that) they can use all of our testing in future releases.

We need to take advantage of the fact that we are a lot of users, for example i can do the testings in Debian 8 and CentOS 7, kvm and openvz boxes.

**Comment by newsyrom** [comment]
we are still waiting for an update and we can help the test in any way that might help

**Comment by vvcares** [comment]
Investing the deve team’s precious manpower into softaculus integration is pointless.

We can bet that,
Not even 5% of server-sys-admin cares about that features.

**Comment by kevzdev** [comment]
Shouldn't the date on the latest update be changed from "Nov 25"  
![before](https://i.imgur.com/TozbLiz.png)  
to  "Nov 25 2016" 
![after](https://i.imgur.com/KTCgIW2.png)  
otherwise people might think (in a week) that VestaCP made an update on Nov 25 2017, wich is not true.

**Comment by anacondaqq** [comment]
They release vesta finally. Closing the issue because it's not actual anymore.
